synth.py

```diff
@@ -0,0 +1,32 @@
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+from pathlib import Path
+import subprocess
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+
+versions = ['v3']
+
+for version in versions:
+    library = gapic.node_library(
+        'monitoring', version,
+        config_path="/google/monitoring/"
```

Comment:
this isn't really going to work for multiple versions so it is odd it is set up to do it.

Reply:
Yes you're right. Setting it to only run for a single version.